app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,38 @@
+class VotesController < ApplicationController
+  before_action :set_review
+  before_action :set_monster
+
+  def handle_vote
+    matching_votes = Vote.where(user_id: current_user.id, review_id: @review.id)
+    vote_value = params[:value]
+    if matching_votes.empty?
+      @vote = Vote.new(user_id: current_user.id, review_id: @review.id, value: vote_value)
+      if @vote.save
```

Comment:
Doesn't make sense in this context where we're only doing assignment in one of the conditions.